packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -2881,6 +2893,27 @@ class StackViewport extends Viewport implements IStackViewport, IImagesLoader {
     };
   }
 
+  /**
+   * Applies the view reference, which may navigate the slice index and apply
+   * other camera modifications
+   */
+  public setView(viewRef?: ViewReference, viewPres?: ViewPresentation): void {
```

Comment:
can we rename to setViewPresentation, since we have getViewPresentation

Reply:
It is a single combined method that can navigate as well as apply a view presentation.  It probably doesn't QUITE work perfectly yet without extra event notifications, but the idea is to only get one set of notifications.